packages/blue-sdk/src/addresses.ts

```diff
@@ -75,6 +75,28 @@ export const addresses = {
     verUsdc: "0x59aaF835D34b1E3dF2170e4872B785f11E2a964b",
     testUsdc: "0xBC77067f829979812d795d516E523C4033b66409",
   },
+  [ChainId.ArbitrumMainnet]: {
+    morpho: "0x6c247b1F6182318877311737BaC0844bAa518F5e",
+    bundler: "0x044aE7bA624a047535F4A6ae5FE6E9EA8d1D93B0",
```

Comment:
This corresponds to `bundler3`, but it should correspond to bundler v2 which is not deployed on Arbitrum... This is blocking as it would introduce unexpected behaviors (most probably 100% failing bundles). Let's wait for bundler3 support coming this week